reporter/otlp_reporter.go

```diff
@@ -235,6 +236,22 @@ func waitGrpcEndpoint(ctx context.Context, cfg *Config) (*grpc.ClientConn, error
 	}
 }
 
+type headerAuth map[string]string
+
+func newHeaderAuth(key, value string) headerAuth {
+	return headerAuth{
```

Comment:
Is there a reason why we don't want to use base64 encoding for basic authentication?

Just thinking about an option like this
https://github.com/johanbrandhorst/grpc-auth-example?tab=readme-ov-file#http-basic-authentication

Reply:
The base64 happens once here:

https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/410/files#diff-cf3f0e395fd57dc7dc3b96b7408a43259afd536516f9b281bea20e976f763002R278

Rather than for every request. 

So the flag would take plain text `user:password` (non base64) a bit like `curl -u user:password`.